core/shared/platform/common/posix/posix_file.c

```diff
@@ -7,6 +7,8 @@
 #include "libc_errno.h"
 #include <unistd.h>
 
+#include "bh_platform.h"
```

Comment:
No need to add this line if the added `convert_errno` in this file is removed, or, keep this file unchanged.